starlette_exporter/middleware.py

```diff
@@ -88,6 +89,7 @@ def __init__(
         app_name: str = "starlette",
         prefix: str = "starlette",
         buckets: Optional[Sequence[Union[float, str]]] = None,
+        group_unhandled_paths: bool = False,
```

Comment:
Can you add the new option to the end? It's unlikely anybody is actually using positional args, but it seems like a general good practice to keep the order the same.

Reply:
yep, updated